.travis.yml

```diff
@@ -1,40 +1,26 @@
-language: c
+language: generic
 os: linux
-sudo: required
-matrix:
```

Comment:
Matrix is for "[build matrices](https://docs.travis-ci.com/user/build-stages/matrix-expansion/)" which let you fire off multiple test configurations at the same time. Since you only want to test on linux with GCC 7, there is no need for a build matrix, and we don't need to use the matrix key.